src/network/requestQueue/index.js

```diff
@@ -42,6 +40,16 @@ module.exports = class RequestQueue {
           queueSize: this.pending.length,
         })
     }
+
+    // FIXME: How do we correctly clearInterval ?
```

Comment:
The `RequestQueue` is only instantiated in one place, so in that place we'd need to call a `requestQueue.destroy()` that calls `clearInterval` if there already is an instance of the request queue.